lib/xandra/connection/utils.ex

```diff
@@ -137,6 +137,7 @@ defmodule Xandra.Connection.Utils do
       # TODO: handle :error frames for things like :protocol_violation.
       case frame do
         %Frame{kind: :ready, body: <<>>} ->
+          Logger.debug("FRAME READY #{inspect(frame)}, #{inspect(rest)}, #{inspect(socket)}}")
```

Comment:
This is already logged one line below, so I don't think we should log this line as well.